proctor-ant-plugin/build.gradle

```diff
@@ -1,3 +1,7 @@
+plugins {
+    id 'com.github.johnrengelman.shadow' version '8.1.1'
```

Comment:
Is this plugin to exclude dependencies that can cause expected issues?

Reply:
I see people are also using in the build.gradle

        compile("com.indeed:proctor-common") {
            exclude group:'org.springframework'
            exclude group:'javax.servlet'
 Is this similar?

Reply:
Shadow jar is not designed that greatly. Need to exclude other libraries or else we include them in our jar which can cause bugs down stream. I confirmed this by checking jar contents using `jar -tf path/to/X.jar `

Reply:
Gotcha.